railties/lib/rails/generators/rails/app/templates/config/environments/production.rb.tt

```diff
@@ -17,6 +17,7 @@ Rails.application.configure do
   # Attempt to read encrypted secrets from `config/secrets.yml.enc`.
   # Requires an encryption key in `ENV["RAILS_MASTER_KEY"]` or
   # `config/secrets.yml.key`.
+  # config.secrets_cipher = "aes-128-gcm"
```

Comment:
I agree. I only erred on including it because existing users of the beta will need to know how to either switch back (bad), or migrate to GCM (better).

There's also the unfortunate case of machines with ancient OpenSSL < 1.0.1, which doesn't support GCM.